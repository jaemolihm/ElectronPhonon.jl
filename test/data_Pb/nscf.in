 &control
    calculation     = 'nscf'
    prefix          = 'pb'
    pseudo_dir      = './'
    outdir          = './temp/'
 /
 &system
    ibrav           = 2
    celldm(1)       = 9.27
    nat             = 1
    ntyp            = 1
    ecutwfc         = 60
    occupations     = 'smearing'
    smearing        = 'mp'
    degauss         = 0.025
 /
 &electrons
    diagonalization = 'david'
    mixing_beta     = 0.7
    conv_thr        = 1.0d-10
 /
ATOMIC_SPECIES
 Pb 207.2   pb_s.UPF
ATOMIC_POSITIONS crystal
Pb        0.000000000   0.000000000   0.000000000
K_POINTS crystal
27
  0.000000000000  0.000000000000  0.000000000000  3.70370370e-02
  0.000000000000  0.000000000000  0.333333333333  3.70370370e-02
  0.000000000000  0.000000000000  0.666666666667  3.70370370e-02
  0.000000000000  0.333333333333  0.000000000000  3.70370370e-02
  0.000000000000  0.333333333333  0.333333333333  3.70370370e-02
  0.000000000000  0.333333333333  0.666666666667  3.70370370e-02
  0.000000000000  0.666666666667  0.000000000000  3.70370370e-02
  0.000000000000  0.666666666667  0.333333333333  3.70370370e-02
  0.000000000000  0.666666666667  0.666666666667  3.70370370e-02
  0.333333333333  0.000000000000  0.000000000000  3.70370370e-02
  0.333333333333  0.000000000000  0.333333333333  3.70370370e-02
  0.333333333333  0.000000000000  0.666666666667  3.70370370e-02
  0.333333333333  0.333333333333  0.000000000000  3.70370370e-02
  0.333333333333  0.333333333333  0.333333333333  3.70370370e-02
  0.333333333333  0.333333333333  0.666666666667  3.70370370e-02
  0.333333333333  0.666666666667  0.000000000000  3.70370370e-02
  0.333333333333  0.666666666667  0.333333333333  3.70370370e-02
  0.333333333333  0.666666666667  0.666666666667  3.70370370e-02
  0.666666666667  0.000000000000  0.000000000000  3.70370370e-02
  0.666666666667  0.000000000000  0.333333333333  3.70370370e-02
  0.666666666667  0.000000000000  0.666666666667  3.70370370e-02
  0.666666666667  0.333333333333  0.000000000000  3.70370370e-02
  0.666666666667  0.333333333333  0.333333333333  3.70370370e-02
  0.666666666667  0.333333333333  0.666666666667  3.70370370e-02
  0.666666666667  0.666666666667  0.000000000000  3.70370370e-02
  0.666666666667  0.666666666667  0.333333333333  3.70370370e-02
  0.666666666667  0.666666666667  0.666666666667  3.70370370e-02
