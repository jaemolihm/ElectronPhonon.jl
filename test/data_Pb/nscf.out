
     Program PWSCF v.6.7MaX starts on  9Sep2021 at 18:54:42 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     2 processors

     MPI processes distributed on     1 nodes
     K-points division:     npool     =       2
     Reading input from nscf.in

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Atomic positions and unit cell read from directory:
     ./temp/pb.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         703     703    211                12387    12387    2109
 


     bravais-lattice index     =            2
     lattice parameter (alat)  =       9.2700  a.u.
     unit-cell volume          =     199.1495 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        14.00
     number of Kohn-Sham states=           11
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     Exchange-correlation= PZ
                           (   1   1   0   0   0   0   0)

     celldm(1)=   9.270000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Pb read from file:
     ./pb_s.UPF
     MD5 check sum: 7774f596943a2055f0d8d19eea6f2ae8
     Pseudo is Norm-conserving, Zval = 14.0
     Generated using "atomic" code by A. Dal Corso  (Quantum ESPRESSO distribution)
     Using radial grid of 1281 points,  2 beta functions with: 
                l(1) =   2
                l(2) =   1

     atomic species   valence    mass     pseudopotential
        Pb            14.00   207.20000     Pb( 1.00)

     48 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Pb  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=    27  Methfessel-Paxton smearing, width (Ry)=  0.0250
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0740741
        k(    2) = (  -0.3333333   0.3333333  -0.3333333), wk =   0.0740741
        k(    3) = (  -0.6666667   0.6666667  -0.6666667), wk =   0.0740741
        k(    4) = (   0.3333333   0.3333333   0.3333333), wk =   0.0740741
        k(    5) = (   0.0000000   0.6666667   0.0000000), wk =   0.0740741
        k(    6) = (  -0.3333333   1.0000000  -0.3333333), wk =   0.0740741
        k(    7) = (   0.6666667   0.6666667   0.6666667), wk =   0.0740741
        k(    8) = (   0.3333333   1.0000000   0.3333333), wk =   0.0740741
        k(    9) = (   0.0000000   1.3333333   0.0000000), wk =   0.0740741
        k(   10) = (  -0.3333333  -0.3333333   0.3333333), wk =   0.0740741
        k(   11) = (  -0.6666667   0.0000000   0.0000000), wk =   0.0740741
        k(   12) = (  -1.0000000   0.3333333  -0.3333333), wk =   0.0740741
        k(   13) = (   0.0000000   0.0000000   0.6666667), wk =   0.0740741
        k(   14) = (  -0.3333333   0.3333333   0.3333333), wk =   0.0740741
        k(   15) = (  -0.6666667   0.6666667  -0.0000000), wk =   0.0740741
        k(   16) = (   0.3333333   0.3333333   1.0000000), wk =   0.0740741
        k(   17) = (   0.0000000   0.6666667   0.6666667), wk =   0.0740741
        k(   18) = (  -0.3333333   1.0000000   0.3333333), wk =   0.0740741
        k(   19) = (  -0.6666667  -0.6666667   0.6666667), wk =   0.0740741
        k(   20) = (  -1.0000000  -0.3333333   0.3333333), wk =   0.0740741
        k(   21) = (  -1.3333333   0.0000000   0.0000000), wk =   0.0740741
        k(   22) = (  -0.3333333  -0.3333333   1.0000000), wk =   0.0740741
        k(   23) = (  -0.6666667  -0.0000000   0.6666667), wk =   0.0740741
        k(   24) = (  -1.0000000   0.3333333   0.3333333), wk =   0.0740741
        k(   25) = (   0.0000000   0.0000000   1.3333333), wk =   0.0740741
        k(   26) = (  -0.3333333   0.3333333   1.0000000), wk =   0.0740741
        k(   27) = (  -0.6666667   0.6666667   0.6666667), wk =   0.0740741

     Dense  grid:    12387 G-vectors     FFT dimensions: (  36,  36,  36)

     Estimated max dynamical RAM per process >       6.49 MB

     Estimated total dynamical RAM >      12.98 MB

     The potential is recalculated from file :
     ./temp/pb.save/charge-density

     Starting wfcs are    9 randomized atomic wfcs +    2 random wfcs

     Band Structure Calculation
     Davidson diagonalization with overlap

     ethr =  7.14E-13,  avg # of iterations = 31.3

     total cpu time spent up to now is        3.4 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000 (  1471 PWs)   bands (ev):

    -5.4835  -5.4835  -5.4835  -5.3264  -5.3264  -0.5406  18.6682  18.6682
    18.6682  21.9665  23.0168

          k =-0.3333 0.3333-0.3333 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k =-0.6667 0.6667-0.6667 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k = 0.3333 0.3333 0.3333 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k = 0.0000 0.6667 0.0000 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-0.3333 1.0000-0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.6667 0.6667 0.6667 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k = 0.3333 1.0000 0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.0000 1.3333 0.0000 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-0.3333-0.3333 0.3333 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k =-0.6667 0.0000 0.0000 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-1.0000 0.3333-0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.0000 0.0000 0.6667 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-0.3333 0.3333 0.3333 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k =-0.6667 0.6667-0.0000 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.3333 0.3333 1.0000 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.0000 0.6667 0.6667 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-0.3333 1.0000 0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-0.6667-0.6667 0.6667 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

          k =-1.0000-0.3333 0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-1.3333 0.0000 0.0000 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-0.3333-0.3333 1.0000 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-0.6667-0.0000 0.6667 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-1.0000 0.3333 0.3333 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k = 0.0000 0.0000 1.3333 (  1580 PWs)   bands (ev):

    -5.6208  -5.5007  -5.2312  -5.2312  -5.2240   2.3298  11.3009  14.2381
    14.2381  18.1257  19.0160

          k =-0.3333 0.3333 1.0000 (  1568 PWs)   bands (ev):

    -5.5465  -5.3719  -5.3199  -5.2919  -5.2209   4.2160   9.4829   9.7281
    14.9713  18.7672  25.8849

          k =-0.6667 0.6667 0.6667 (  1540 PWs)   bands (ev):

    -5.5070  -5.4540  -5.4540  -5.2369  -5.2369   1.6122   9.7712  17.2451
    17.2451  20.9130  22.5753

     the Fermi energy is     9.8434 ev
     (compare with:    11.3616 eV, computed in scf)

     Writing output data file ./temp/pb.save/
 
     init_run     :      0.09s CPU      0.10s WALL (       1 calls)
     electrons    :      2.92s CPU      2.95s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.01s CPU      0.01s WALL (       1 calls)
     hinit0       :      0.07s CPU      0.08s WALL (       1 calls)

     Called by electrons:
     c_bands      :      2.92s CPU      2.95s WALL (       1 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      14 calls)
     cegterg      :      2.56s CPU      2.59s WALL (      27 calls)

     Called by *egterg:
     cdiaghg      :      0.06s CPU      0.07s WALL (     442 calls)
     h_psi        :      2.44s CPU      2.46s WALL (     469 calls)
     g_psi        :      0.03s CPU      0.03s WALL (     428 calls)

     Called by h_psi:
     h_psi:calbec :      0.02s CPU      0.02s WALL (     469 calls)
     vloc_psi     :      2.40s CPU      2.41s WALL (     469 calls)
     add_vuspsi   :      0.02s CPU      0.02s WALL (     469 calls)

     General routines
     calbec       :      0.02s CPU      0.02s WALL (     469 calls)
     fft          :      0.00s CPU      0.01s WALL (       3 calls)
     fftw         :      2.14s CPU      2.16s WALL (    5280 calls)
     davcio       :      0.00s CPU      0.00s WALL (      28 calls)
 
     Parallel routines
 
     PWSCF        :      3.27s CPU      3.39s WALL

 
   This run was terminated on:  18:54:46   9Sep2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
